packages/theme/src/components/list.ts

```diff
@@ -0,0 +1,18 @@
+const parts = ["container", "item", "icon"]
+
+const baseStyleContainer = {}
```

Comment:
Leaving these empty seams weird, but it does give us auto completion for the theme.